### PR TITLE
tutorial level instructions

### PR DESCRIPTION
**Pull request by rachel92124**
I added instructions in level 0 that appear based on where you are in the maze
LMK if there are instructions I should add/change. I'm not sure if I am specific enough in my instructions.

### CODE REVIEW

File: Unity/Assets/Scripts/TutorialInstructions.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    {
+        //InstrUI.SetActive(false); // Set game object to not active
+        instrText.text = "";
+    }
+
+    private void OnControllerColliderHit(ControllerColliderHit hit)
+    {
+        if (hit.collider.name == "MoveInstrCollider")
+        {
+            Show("To move forward use the arrow keys, WASD, or tilt the cotroller forward");
```
Comment (sydneylaiw) [inline Unity/Assets/Scripts/TutorialInstructions.cs]: small typo: controller

**Comment by sydneylaiw** [comment]
Other than that, LGTM. Only thing is if possible, it might be nice to make the text bolder/in a diff color so it pops more against the maze background.

**Comment by anreeta** [comment]
Tutorial level looks neat! Instead of making text a different color you could put the text on a Panel UI element with a semi-transparent background to make it stand out regardless of where in the level you are

Also, very minor but for some of the instructions, you could put a comma between after the "To jump/duck/etc" part

File: Unity/Assets/Scripts/TutorialInstructions.cs [hunk 2]
```diff
@@ -10,33 +10,33 @@ public class TutorialInstructions : MonoBehaviour
...
         }
         else if (hit.collider.name == "TurnInstrCollider")
         {
-            Show("To turn use 'z' and 'x' or tilt the controller left and right");
+            Show("To turn, use 'z' and 'x' or tilt the controller left and right");
         }
         else if (hit.collider.name == "JumpInstrCollider")
         {
-            Show("To jump press the space bar or lighly jump while in frame of your webcam");
+            Show("To jump, press the space bar or lighly jump while in frame of your webcam");
```
Comment (sydneylaiw) [inline Unity/Assets/Scripts/TutorialInstructions.cs]: *lightly, sorry didn't notice this the first time

**Review by sydneylaiw — CHANGES_REQUESTED** [review]

File: Unity/Assets/Scripts/TutorialInstructions.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            Show("To move forward, use the arrow keys, WASD, or tilt the cotroller forward");
+        }
+        else if (hit.collider.name == "DuckInstrCollider")
+        {
+            Show("To duck, hold shift or duck while in frame of your webcam");
+        }
+        else if (hit.collider.name == "TurnInstrCollider")
+        {
+            Show("To turn, use 'z' and 'x' or tilt the controller left and right");
```
Comment (anreeta) [inline Unity/Assets/Scripts/TutorialInstructions.cs]: Very minor but maybe can say "To turn left and right, use 'z' and 'x' or tilt the controller"

Likely also fine as is

**Comment by anreeta** [comment]
Have looked at the PR again. The semi-transparent background looks fine to me. 

One minor thing: there are apparently inconsistent line endings in TutorialInstructions.cs... maybe want to make them all LF?

Otherwise, after Sydney's changes, it looks good to merge

**Pull request merged**